GTG/gtk/editor/editor.py

```diff
@@ -381,22 +451,22 @@ def refresh_editor(self, title=None, refreshtext=False):
                 mi.connect("activate", self.inserttag, tagname)
                 mi.show()
                 menu.append(mi)
-        if tag_count > 0:
-            self.inserttag_button.set_menu(menu)
+        # if tag_count > 0:
+        #      self.inserttag_button.set_menu(menu)
```

Comment:
I don't think this code will be needed, you will show the popover menu instead anyway